pandas/tests/extension/test_interval.py

```diff
@@ -95,6 +95,12 @@ def test_EA_types(self, engine, data):
     def test_invert(self, data):
         super().test_invert(data)
 
+    @pytest.mark.xfail(
+        reason="IntervalArray.round is not implemented."
```

Comment:
i wouldn't expect this to ever work, would you?

Reply:
I wasn't familiar with IntervalArrays. I've changed the message.